app/src/main/res/values/strings.xml

```diff
@@ -3,7 +3,9 @@
     <string name="app_name">Connect</string>
 
     <!--ENDPOINTS-->
-    <string name="api_url">https://gaia.blockstack.org/</string>
+    <string name="gaia_endpoint">https://gaia.blockstack.org/</string>
+    <string name="hub_endpoint">https://hub.blockstack.org/</string>
+    <string name="registrar_endpoint">https://registrar.blockstack.org/</string>
     <string name="about_us_url">https://www.blockstack.org/how-blockstack-works</string>
     <string name="newsletter_url">http://newsletters.blockstack.org/</string>
     <string name="discord_url">https://discord.gg/tR8HjHnb</string>
```

Comment:
URLs could go into their own `urls.xml` file, and they all should have a `translable=false` flag.